src/styles/app/03-app-shell.css

```
@@ -34,7 +37,11 @@
 
 :root[data-translucent-app-frame="false"] .appShell {
 	background: var(--bg-primary);
-	border-color: var(--border-light);
+	--app-frame-inline: 0px;
+	--app-frame-block: 0px;
+	border-width: 0;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)